.github/workflows/continuous-integration-dotnet.yml

```diff
@@ -2,26 +2,16 @@ name: SonarCloud scanning and coverage generation
 
 on:
   push:
-    branches:
-      - master
-    paths:
-      - 'DFE.*/**'
-      - '!CypressTests/**'
-  pull_request:
     branches: [ master ]
-    types: [ opened, synchronize, reopened ]
-    paths:
-      - 'DFE.*/**'
-      - '!CypressTests/**'
+  pull_request:
```

Comment:
Supposed to be empty?

Reply:
Yep this is the syntax for triggering on _any_ pull request regardless of type or path